components/scream/src/control/atmosphere_driver.cpp

```diff
@@ -146,7 +153,7 @@ void AtmosphereDriver::initialize_output_manager () {
     m_output_manager.set_params(out_params);
     m_output_manager.set_comm(m_atm_comm);
     m_output_manager.set_grids(m_grids_manager);
-    m_output_manager.set_repo(m_field_repo);
+    m_output_manager.set_fm(get_ref_grid_field_mgr());
```

Comment:
Consider `set_field_mgr` or `set_field_manager` here. Again, no strong feelings--it's probably fine as is.